test/translator/text-translator.js

```diff
@@ -1,3 +1,3 @@
-module.exports = function (text) {
-    return "module.exports = " + JSON.stringify(text) + ";";
+module.exports = function (module) {
+    module.text = "module.exports = " + JSON.stringify(module.text) + ";";
 };
```

Comment:
These reflect the substantiative changes. Translators and optimizers accept the module object, modify it, and may return a promise. This makes the signatures of translators, optimizers, and compilers more uniform.